kanidm_client/src/asynchronous.rs

```diff
@@ -829,6 +829,17 @@ impl KanidmAsyncClient {
             .await
     }
 
+    pub async fn idm_account_add_attr(
+        &self,
+        id: &str,
+        attr: &str,
+        values: &[&str],
+    ) -> Result<bool, ClientError> {
+        let m: Vec<_> = values.iter().map(|v| (*v).to_string()).collect();
```

Comment:
Changed m to msg. 

Yes the Vec<_> hint is needed. Because self.perform_post_request takes `R: Serialise`, without the hint here .collect doesnt know what to collect into. I could use the turbofish syntax, but honestly, I always forget it. I think it's .collect::Vec<_>() but I think that's a bit less clear than `let msg: Vec<_> = ... `